web/src/main/webapp/my-app/marketplace/controllers.js

```diff
@@ -57,6 +57,16 @@ define(['angular', 'jquery', 'require'], function(angular, $, require) {
           && portlet.portletName // there is a portletName
           && portlet.portletName.indexOf('cms') != -1; // static content portlet
       };
+
+      var isGuest = function() {
```

Comment:
:art:

```js
var isGuest = mainService.isGuest;
```

Reply:
This defines a function `isGuest`, that leverages `mainService.isGuest`, and resolves the `result`.